.github/workflows/swift.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Should we have a workflow that runs tests on PRs too?

Reply:
It ran on my PR?

Reply:
But I wonder if main is correct - I think this one uses master right?

Reply:
Ya I was surprised the repo used main, master would be in the case for most older repos like this, right? in the Android SDK we have two workflows, one runs on master, the other runs on feature branches

Reply:
I'll fix to use master. I think I'd rather just run on PRs vs branches right?

Reply:
Ya just on PRs makes sense to me! Thats what I meant sorry